ompi/mca/pml/ob1/pml_ob1_isend.c

```diff
@@ -30,6 +30,11 @@
 #include "ompi/peruse/peruse-internal.h"
 #include "ompi/runtime/ompi_spc.h"
 
+#if MPI_VERSION >= 4
+extern int ompi_pml_base_deprecation_warn_level;
+extern ompi_request_t ompi_request_empty_send;
```

Comment:
Extern isn't right here, or this should be moved into a header if defined somewhere else.